secboot/secboot_tpm.go

```diff
@@ -188,11 +188,34 @@ func MeasureSnapModelWhenPossible(findModel func() (*asserts.Model, error)) erro
 	return nil
 }
 
-// LockTPMSealedKeys manually locks access to the sealed keys. Meant to be
+// LockSealedKeys manually locks access to the sealed keys. Meant to be
 // called in place of passing lockKeysOnFinish as true to
```

Comment:
that flag doesn't exist anymore so probably a good time to review comments here mentioning it. Here we need to to change the comment to be something like "must always be called in early boot , either after having successfully unlocked keys or as early as possible if that wasn't the case. This to prevent for example alternatively booted user space (e.g. from external storage) to access the keys."